net/sslh/src/opnsense/service/templates/OPNsense/Sslh/sslh.jinja

```diff
@@ -0,0 +1,11 @@
+{# /usr/local/etc/rc.conf.d/sslh #}
+{%  set plugin_name = 'sslh' %}
+{%  if OPNsense[plugin_name] is defined %}{# Prevent configd reload from erroring out if node doesn't exist #}
+{%      set cfg = OPNsense[plugin_name] %}
+{%      if cfg['enabled'] is defined %}
```

Comment:
could be reduced, but not required

there's an "empty" helper that would  make sure the config array is populated before access

Reply:
Thanks, I'll do that. I always forget about those helper functions.